### PR TITLE
User guide tests and remove manual steps for plugging in CA cert

### PR DESCRIPTION
**Pull request by lei-tang**
Please provide a description for what this PR is for: 
- Add user guide tests for plugging in CA cert.
- Remove the manual steps in the user guide of plugging in CA cert to make it easier for an user to try the guide.

### CODE REVIEW

**Comment by lei-tang** [comment]
@frankbu Thanks! I have converted the tests to work the new way. Please take another look.

File: scripts/snip.py [hunk 1]
```diff
@@ -91,7 +91,7 @@
             else:
                 id = "snip_%s_%d" % (section, snipnum)
             if kind == "bash":
-                script = "\n%s() {\n" % id
+                script = "\n# shellcheck disable=SC2046\n%s() {\n" % id
```
Comment (frankbu) [inline scripts/snip.py:94]: This is wrong. We should not be disabling any lint checks in generated bash functions. If there are lint errors in the generated snip.sh functions, it means that the command in the original index.md file is not following bash best practice. This is another benefit of the new model: we get lint checking of the commands that we are publishing in the docs.

You should fix the lint problem by changing the command in index.md and then regen.

Comment (lei-tang) [reply scripts/snip.py]: Done: the lint problem is fixed by changing the command in index.md and the disabling lint check has been removed.

**Pull request merged**